client/my-sites/checkout/concierge-session-nudge/concierge-support-session/index.jsx

```diff
@@ -0,0 +1,221 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Fragment, PureComponent } from 'react';
+import Gridicon from 'gridicons';
+import formatCurrency from '@automattic/format-currency';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Button from 'components/button';
+import DocumentHead from 'components/data/document-head';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export class ConciergeSupportSession extends PureComponent {
+	render() {
+		const { receiptId, translate } = this.props;
+
+		const title = translate( 'Checkout ‹ Support Session', {
+			comment: '"Checkout" is the part of the site where a user is preparing to make a purchase.',
+		} );
+
+		return (
+			<>
+				<PageViewTracker path="/checkout/:site/offer-support-session/:receipt_id" title={ title } />
+				<DocumentHead title={ title } />
+				{ receiptId ? (
+					<CompactCard className="concierge-support-session__card-header">
+						{ this.header() }
+					</CompactCard>
+				) : (
+					''
+				) }
+				<CompactCard className="concierge-support-session__card-body">{ this.body() }</CompactCard>
+				<CompactCard className="concierge-support-session__card-footer">
+					{ this.footer() }
+				</CompactCard>
+			</>
+		);
+	}
+
+	header() {
+		const { translate } = this.props;
+		return (
+			<header className="concierge-support-session__header">
+				<h2 className="concierge-support-session__title">
+					{ translate( 'A special one time offer, just for you.' ) }
+				</h2>
+			</header>
+		);
+	}
+
+	body() {
+		const { translate, productCost, productDisplayCost, currencyCode } = this.props;
+		// Full cost should be 150% of base cost
+		const fullCost = Math.round( productCost * 1.5 );
+		const savings = fullCost - productCost;
+		return (
+			<Fragment>
```

Comment:
Since we have introduced `<>` syntax above already, I'd suggest us to replace `<Fragment>` with that overall.

Reply:
Done in [a5da8b9](https://github.com/Automattic/wp-calypso/pull/33996/commits/a5da8b95b9c91a397e4e77aeaeb2c84c8451fbe5).